src/os/control/rotatecontrol.js

```diff
@@ -53,7 +53,8 @@ os.control.Rotate.render = function(mapEvent) {
       return;
     }
 
-    rotation = camera.getHeading();
+    // OpenLayers rotation is in the opposite direction of WebGL camera heading.
+    rotation = -camera.getHeading();
```

Comment:
Can/should this negative multiplier be renderer-based, or is this pretty much standard for a "heading" versus the corresponding projection transformation?

Reply:
Internally the renderer can do what it wants, but the API should be standardized so switching renderers doesn't change inputs/outputs.